ibm/service/iampolicy/resource_ibm_iam_action_control_template_version_test.go

```diff
@@ -0,0 +1,173 @@
+					testAccCheckIBMActionControlTemplateExists("ibm_iam_action_control_template.action_control_template", actConf),
+					resource.TestCheckResourceAttr("ibm_iam_action_control_template.action_control_template", "name", actName),
+					resource.TestCheckResourceAttr("ibm_iam_action_control_template_version.action_control_template_version", "name", actName),
+					resource.TestCheckResourceAttr("ibm_iam_action_control_template.action_control_template", "action_control.0.actions.0", actCreateAction),
+					resource.TestCheckResourceAttr("ibm_iam_action_control_template_version.action_control_template_version", "action_control.0.actions.#", "2"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccIBMIAMActionControlVersionTemplateUpdateCommit(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { acc.TestAccPreCheck(t) },
+		Providers:    acc.TestAccProviders,
+		CheckDestroy: testAccCheckIBMActionControlTemplateDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckIBMActionControlTemplateVersionConfig(actName, actCreateAction, actDeleteAction),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckIBMActionControlTemplateExists("ibm_iam_action_control_template.action_control_template", actConf),
+					resource.TestCheckResourceAttr("ibm_iam_action_control_template.action_control_template", "name", actName),
+					resource.TestCheckResourceAttr("ibm_iam_action_control_template_version.action_control_template_version", "name", actName),
+					resource.TestCheckResourceAttr("ibm_iam_action_control_template.action_control_template", "action_control.0.actions.0", actCreateAction),
+					resource.TestCheckResourceAttr("ibm_iam_action_control_template_version.action_control_template_version", "action_control.0.actions.0", actDeleteAction),
+				),
+			},
+			{
+				Config: testAccCheckIBMActionControlTemplateVersionUpdateConfig(actName, actCreateAction, actCreateAction, actDeleteAction, true),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckIBMActionControlTemplateExists("ibm_iam_action_control_template.action_control_template", actConf),
+					resource.TestCheckResourceAttr("ibm_iam_action_control_template.action_control_template", "name", actName),
+					resource.TestCheckResourceAttr("ibm_iam_action_control_template_version.action_control_template_version", "name", actName),
+					resource.TestCheckResourceAttr("ibm_iam_action_control_template.action_control_template", "action_control.0.actions.0", actCreateAction),
+					resource.TestCheckResourceAttr("ibm_iam_action_control_template_version.action_control_template_version", "action_control.0.actions.#", "2"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckIBMActionControlTemplateVersionConfig(name string, actionControl string, actionControl1 string) string {
+	return fmt.Sprintf(`
+		resource "ibm_iam_action_control_template" "action_control_template" {
+			name = "%s"
+			description = "Base template Testing"
```

Comment:
Similar to above, we might want to note these templates are created for template_version testing, and note these should be cleaned up and which test the relate to?
I'm fine leaving all of these as they are, but might be helpful if there's any issue.